src/main/java/com/fouristhenumber/utilitiesinexcess/ModBlocks.java

```diff
@@ -83,18 +83,42 @@ public enum ModBlocks {
-    LAVA_GENERATOR(GeneratorConfig.enableLavaGenerator, new BlockLavaGenerator("lava_generator"), "lava_generator"),
-    ENDER_GENERATOR(GeneratorConfig.enableEnderGenerator, new BlockEnderGenerator("ender_generator"), "ender_generator"),
-    REDSTONE_GENERATOR(GeneratorConfig.enableRedstoneGenerator, new BlockRedstoneGenerator("redstone_generator"), "redstone_generator"),
-    FOOD_GENERATOR(GeneratorConfig.enableFoodGenerator, new BlockFoodGenerator("food_generator"), "food_generator"),
-    POTION_GENERATOR(GeneratorConfig.enablePotionGenerator, new BlockPotionGenerator("potion_generator"), "potion_generator"),
-    SOLAR_GENERATOR(GeneratorConfig.enableSolarGenerator, new BlockSolarGenerator("solar_generator"), "solar_generator"),
-    TNT_GENERATOR(GeneratorConfig.enableTNTGenerator, new BlockTNTGenerator("tnt_generator"), "tnt_generator"),
-    PINK_GENERATOR(GeneratorConfig.enablePinkGenerator, new BlockPinkGenerator("pink_generator"), "pink_generator"),
-    NETHER_STAR_GENERATOR(GeneratorConfig.enableNetherStarGenerator, new BlockNetherStarGenerator("nether_star_generator"), "nether_star_generator"),
+    LOW_TEMPERATURE_FURNACE_GENERATOR(GeneratorConfig.enableLowTemperatureFurnaceGenerator, new BlockLowTemperatureFurnaceGenerator("low_temperature_furnace_generator", 1), "low_temperature_furnace_generator"),
+    LOW_TEMPERATURE_FURNACE_GENERATOR_PLUS(GeneratorConfig.enableLowTemperatureFurnaceGenerator, new BlockLowTemperatureFurnaceGenerator("low_temperature_furnace_generator_plus", 8), "low_temperature_furnace_generator_plus"),
+    LOW_TEMPERATURE_FURNACE_GENERATOR_PLUSPLUS(GeneratorConfig.enableLowTemperatureFurnaceGenerator, new BlockLowTemperatureFurnaceGenerator("low_temperature_furnace_generator_plusplus", 64), "low_temperature_furnace_generator_plusplus"),
+    FURNACE_GENERATOR(GeneratorConfig.enableFurnaceGenerator, new BlockFurnaceGenerator("furnace_generator", 1), "furnace_generator"),
+    FURNACE_GENERATOR_PLUS(GeneratorConfig.enableFurnaceGenerator, new BlockFurnaceGenerator("furnace_generator_plus", 8), "furnace_generator_plus"),
+    FURNACE_GENERATOR_PLUSPLUS(GeneratorConfig.enableFurnaceGenerator, new BlockFurnaceGenerator("furnace_generator_plusplus", 64), "furnace_generator_plusplus"),
+    HIGH_TEMPERATURE_FURNACE_GENERATOR(GeneratorConfig.enableHighTemperatureFurnaceGenerator, new BlockHighTemperatureFurnaceGenerator("high_temperature_furnace_generator", 1), "high_temperature_furnace_generator"),
+    HIGH_TEMPERATURE_FURNACE_GENERATOR_PLUS(GeneratorConfig.enableHighTemperatureFurnaceGenerator, new BlockHighTemperatureFurnaceGenerator("high_temperature_furnace_generator_plus", 8), "high_temperature_furnace_generator_plus"),
+    HIGH_TEMPERATURE_FURNACE_GENERATOR_PLUSPLUS(GeneratorConfig.enableHighTemperatureFurnaceGenerator, new BlockHighTemperatureFurnaceGenerator("high_temperature_furnace_generator_plusplus", 1), "high_temperature_furnace_generator_plusplus"),
```

Comment:
Shouldn't this have a 64x multiplier?
```suggestion
    HIGH_TEMPERATURE_FURNACE_GENERATOR_PLUSPLUS(GeneratorConfig.enableHighTemperatureFurnaceGenerator, new BlockHighTemperatureFurnaceGenerator("high_temperature_furnace_generator_plusplus", 64), "high_temperature_furnace_generator_plusplus"),
```